tests/fuzz/s2n_client_key_recv_fuzz_test.c

```diff
@@ -29,6 +29,7 @@
 
 #include "api/s2n.h"
 #include "stuffer/s2n_stuffer.h"
+#include "tls/s2n_cipher_suites.h"
```

Comment:
With no other changes, is this addition needed?

Reply:
Yes. The fuzz test was getting its s2n_cipher_suites dependency met through another header file that included s2n_cipher_suites.h, but no longer does. Our includes are a bit of a mess :\